archetypes/default.md

```diff
@@ -1,4 +1,5 @@
 ---
 title: "{{ replace .Name "-" " " | title }}"
 date: {{ .Date }}
----
```

Comment:
I don't think making this the default is the best idea. How do you feel about not adding a default for this field when not specified?

Reply:
Ah okay, I misread the context of what this was for because of the section of the README that mentions where the post archetype params can be seen, although that was for _index.md and not this, which is defaults. I can revert this part of the change, but I'd also like to note that afaik there is no mention of this param within the README and didn't even know it was a thing until recently. Maybe it would be worth adding it there or something like an example.md and referencing that.